src/main/kotlin/com/timcastelijns/room15bot/bot/eventhandlers/MessageEventHandler.kt

```diff
@@ -116,6 +117,14 @@ class MessageEventHandler(
         }
     }
 
+    private fun processBenzCommand(user: User) {
+        if (user.id == 4467208L) {
+            actor.acceptMessage(messageFormatter.asBenzString())
```

Comment:
use replies here so it's extra clear who is a peasant and who is not. You have to pass the messageId to this fun along with the user then.